.gitignore

```diff
@@ -184,6 +184,81 @@ UpgradeLog*.htm
 # Microsoft Fakes
 FakesAssemblies/
 
+# Covers JetBrains IDEs: CLion
+# Reference: https://intellij-support.jetbrains.com/hc/en-us/articles/206544839
+
+# Ignore entire idea
+# Comment as required
+*.idea
+
+# User-specific stuff
```

Comment:
what about just saying: `.idea`?

Reply:
For CLion I tend to use:

```
.idea/*
cmake-build-debug/*
cmake-build-release/*
```

Reply:
git doc says to use `.idea/` etc. Please check